app/ts/components/subcomponents/ChainSelector.tsx

```diff
@@ -42,7 +31,7 @@ export function ChainSelector(params: ChainSelectorParams) {
 		<div class = 'dropdown-menu' id = 'dropdown-menu' role = 'menu' style = 'right: -10px; min-width: 160px; left: unset'>
 			<div class = 'dropdown-content'>
 				{
-					rpcEntries === undefined ? <></> : rpcEntries.map((rpcEntry) => { return (
+					params.rpcEntries.map((rpcEntry) => { return (
 						<a href = '#' class = { `dropdown-item ${ rpcEntry.httpsRpc === params.rpcNetwork.httpsRpc ? 'is-active' : '' }` } onClick = { () => changeRpc(rpcEntry) } >
```

Comment:
this feels like a good place for a button element instead of anchor

Reply:
hmm, you feel the styling of button would be better than anchors?